crates/solver/src/driver.rs

```diff
@@ -266,6 +269,12 @@ impl Driver {
             prices: auction.prices.clone(),
         };
 
+        if !self.process_partially_fillable_orders {
+            auction
+                .orders
+                .retain(|order| !order.data.partially_fillable);
```

Comment:
We also need to check that they are limit orders right? We still want partially fillable liquidity orders AFAIU.